Fix testing documentation for integration tests (#987)
sbt/website: src/reference/02-DetailTopics/02-Configuration/14-Testing.md

    test classpath

> The standard testing tasks are available, but must be prefixed with
> `it:`. For example to run all integration tests:

Reviewer: ```suggestion
`IntegrationTest/`. For example to run all integration tests:
```

Author: Fixed

```

> > it:test

Reviewer: ```suggestion
> IntegrationTest/test
```

Author: Fixed

```

Or to run a specific test:

```

> > it:testOnly org.example.AnIntegrationTest

Reviewer: ```suggestion
> IntegrationTest/testOnly org.example.AnIntegrationTest
```

Author: Fixed
